src/request/index.ts

```diff
@@ -243,6 +243,10 @@ export function jsonRequest(
 		header.context.csrfToken = requestOptions.csrfToken;
 	}
 
+	if (requestOptions.authToken) {
+		options.headers['Cookie'] = `ZM_AUTH_TOKEN=${requestOptions.authToken}`;
```

Comment:
this should not happen in case of browsers

Reply:
Right. `requestOptions.authToken` will always be `undefined` except for offline-sync where we are passing explicitly this with value.